pkg/koordlet/statesinformer/states_informer.go

```diff
@@ -209,3 +226,29 @@ func (s *statesInformer) setupInformers() {
 	s.setupNodeInformer()
 	s.setupNodeSLOInformer()
 }
+
+func newKubeletStub(node *corev1.Node, addressPreferred string, timeout int) (KubeletStub, error) {
+	var address string
+	var err error
+	addressPreferredType := corev1.NodeAddressType(addressPreferred)
+	// if the address of the specified type has not been set or error type, InternalIP will be used.
+	if addressPreferredType == "" ||
+		(addressPreferredType != corev1.NodeHostName &&
+			addressPreferredType != corev1.NodeExternalIP &&
+			addressPreferredType != corev1.NodeExternalDNS &&
+			addressPreferredType != corev1.NodeInternalIP &&
+			addressPreferredType != corev1.NodeInternalDNS) {
+		klog.Warningf("Wrong address type or empty type, InternalIP will be used, error: (%+v).", addressPreferredType)
+		addressPreferredType = corev1.NodeInternalIP
+	}
+	address, err = util.GetNodeAddress(node, addressPreferredType)
+	if err != nil {
+		klog.Fatalf("Get node address error: %v type(%s) ", err, addressPreferred)
+	}
+	token, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/token")
```

Comment:
let's declare `/var/run/secrets/kubernetes.io/serviceaccount/token` in a const

Reply:
the serviceaccount  can used to access kubelet https port？